packages/react-console/src/components/AccessConsoles/AccessConsoles.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+
+import constants from '../common/constants';
+
+import '@patternfly/react-console/css/AccessConsoles.css';
```

Comment:
Now that these are CSS, can you move them to `react-styles/src/css/components`? That way you can get types and remove the copy step.